sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/DocumentServiceLeaseUpdaterImpl.java

```diff
@@ -87,7 +86,7 @@ public DocumentServiceLeaseUpdaterImpl(ChangeFeedContextClient client) {
                             serverLease.getConcurrencyToken());
                         arrayLease[0] = serverLease;
 
-                        throw Exceptions.propagate(new RuntimeException(""));
+                        throw new RuntimeException("");
```

Comment:
please add a message to the RuntimeException.

Reply:
Fixed